content/templates/main/change_language.html

```diff
@@ -0,0 +1,33 @@
+{% extends 'layouts/default/page.html' %}
+
+{% load i18n %}
+
+{% block content %}
+
+    {% get_current_language as language_code %}
+    {% get_available_languages as languages %}
+
+    <h4>{% trans 'Change language' %}</h4>
```

Comment:
Are multiple languages part of your MVP? That is generally considered an advanced feature.